README.md

```diff
@@ -52,6 +97,31 @@ $ npm run server
 
 and navigate to: http://127.0.0.1:3001/test/func/fixtures/
 
+### Releases
+
+**IMPORTANT - NPM**: To correctly run `preversion` your first step is to make
```

Comment:
@baer -- It's a specific version of v2, but I'd rather everyone be on v3 anyways, so `npm install -g npm` is recommended.